aws/resource_aws_codebuild_project_test.go

```diff
@@ -1683,24 +1694,19 @@ func TestAccAWSCodeBuildProject_SecondaryArtifacts_ArtifactIdentifier(t *testing
 	bName := acctest.RandomWithPrefix("tf-acc-test-bucket")
 	resourceName := "aws_codebuild_project.test"
 
-	artifactIdentifier1 := "artifactIdentifier1"
-	artifactIdentifier2 := "artifactIdentifier2"
```

Comment:
Nit: i think you could actually keep these since they're re-used in the test steps :) unless there's another intention for removing?